src/components/organize/tasks/TaskAssigneesList/index.tsx

```diff
@@ -16,13 +16,10 @@ const TaskAssigneesList: React.FunctionComponent<{ assignedTasks: ZetkinAssigned
         if (first.status === ASSIGNED_STATUS.ASSIGNED && second.status === ASSIGNED_STATUS.IGNORED) return 1;
         if (first.status === ASSIGNED_STATUS.IGNORED && second.status === ASSIGNED_STATUS.ASSIGNED) return -1;
 
-        const firstDate = first.completed ? dayjs(first.completed) : dayjs(first.assigned);
-        const secondDate = second.completed ? dayjs(second.completed) : dayjs(second.assigned);
+        const firstDate = first.completed ? dayjs(first.completed) : first.ignored ? dayjs(first.ignored) : dayjs(first.assigned);
+        const secondDate = second.completed ? dayjs(second.completed) : second.ignored ? dayjs(second.ignored) : dayjs(second.assigned);
```

Comment:
I'm not a fan of nested ternaries. Is there another way that this could be expressed that would be easier to read?